bin/run_all.sh

```diff
@@ -35,7 +35,7 @@ for benchmark in `cat $root_dir/conf/benchmarks.lst`; do
     if [ $? -ne 0 ]
     then
 	echo "ERROR: ${benchmark} prepare failed!"
-        continue
+        exit $?
```

Comment:
@carsonwang Sorry for this mistake, you are right. I'll test as you say right now.